src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/GovNotifyEmailService.kt

```diff
@@ -26,6 +26,9 @@ class GovNotifyEmailService(
       is AmendedCourtBookingEmail -> send(email, emailTemplates.amendedCourtBookingOwner)
       is AmendedCourtBookingPrisonCourtEmail -> send(email, emailTemplates.amendedCourtBookingPrisonCourtEmail)
       is AmendedCourtBookingPrisonNoCourtEmail -> send(email, emailTemplates.amendedCourtBookingPrisonCourtNoEmail)
+      is CancelledCourtBookingEmail -> send(email, emailTemplates.cancelledCourtBookingOwner)
+      is CancelledCourtBookingPrisonCourtEmail -> send(email, emailTemplates.cancelledCourtBookingPrisonCourtEmail)
+      is CancelledCourtBookingPrisonNoCourtEmail -> send(email, emailTemplates.cancelledCourtBookingPrisonCourtNoEmail)
```

Comment:
Just noticed a slight name diff email class ends with `NoCourtEmail` and template ends with `CourtNoEmail.` Same for amend, meh.